src/games/strategy/triplea/ai/proAI/data/ProMyMoveOptions.java

```diff
@@ -56,4 +59,8 @@ public ProMyMoveOptions(final ProMyMoveOptions myMoveOptions) {
     return transportList;
   }
 
+  public Map<Unit, Set<Territory>> getBomberMoveMap() {
```

Comment:
So the problem is if you move all the code that is touching TerritoryManager variables into it you'll have 1 class for most of the AI with 10000+ lines. TerritoryManager is representing the AIs view of the map. It tracks which units can move where and where the AI chooses to move units. In this case, it actually does makes sense to have what is primarily a data object to track that state.